calculator/src/components/GrowthInput.js

```diff
@@ -1,57 +1,85 @@
 import React,{useState} from "react";
-import "./styles.scss"
+import {Bar} from 'react-chartjs-2';
+import {Form} from 'react-bootstrap'
 
 export default function GrowthInput(props) {
```

Comment:
I'd have to agree with jcarrera94 here... I don't see a use for props in this code later on, and it is not inherently required. Removing _props_  is just one simple way to clean up your code for others. 

If you are intending to use props at a later iteration of this project, I'd suggest inserting a comment to that effect to reduce any confusion.

Reply:
sorry I copy-pasted the from my other project to get the structure and forgot to remove props.